projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -1603,16 +1627,277 @@ public static function wrap_message_in_html_tags( $body ) {
 			str_replace(
 				"\t",
 				'',
-				'<!doctype html>
-				<html xmlns="http://www.w3.org/1999/xhtml">
-				<body>
-
-				%s
-
-				</body>
-				</html>'
+				apply_filters(
+					'jetpack_forms_respone_email_template',
+					'
+					<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```

Comment:
Maybe that template could be moved to its own file instead of hardcoding it here, and folks would be able to point to their own template instead?